UIDeviceIdentifier/UIDeviceHardware.m

```diff
@@ -105,6 +106,18 @@ + (NSString *) platformString
             @"iPad7,4": @"iPad Pro 10.5-inch (WiFi/Cellular)",
             @"iPad7,5": @"iPad 6 (WiFi)",
             @"iPad7,6": @"iPad 6 (WiFi/Cellular)",
+            @"iPad8,1": @"iPad Pro 11-inch 3rd-gen (WiFi)",
+            @"iPad8,2": @"iPad Pro 11-inch 3rd-gen (WiFi)",
```

Comment:
Every second identifier is for the 1TB version. I did not include this detail in the platform string.

Reply:
Thats fine 👍